processing/src/main/java/io/druid/query/topn/AlphaNumericTopNMetricSpec.java

```diff
@@ -46,7 +46,7 @@ public AlphaNumericTopNMetricSpec(
   @Override
   public Comparator getComparator(List<AggregatorFactory> aggregatorSpecs, List<PostAggregator> postAggregatorSpecs)
   {
-    return new AlphanumComparator();
+    return ComparatorUtils.NATURAL_COMPARATOR;
```

Comment:
there doesnt seem to be an alphanum impl in guava or commons, 
probably we will hv to write one ourselves